config/webpack.common.js

```diff
@@ -2,17 +2,17 @@
  * @author: @AngularClass
  */
 
-var webpack = require('webpack');
-var helpers = require('./helpers');
+const webpack = require('webpack');
+const helpers = require('./helpers');
```

Comment:
I updated the repo to use it
https://github.com/AngularClass/angular2-webpack-starter/commit/2a155cbedde9fc289fb4032ff6cbf7f1e111e289
can you rebase so we can get this merged